serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-status-conditions.adoc

```diff
@@ -84,13 +84,13 @@ The following table lists the possible Conditions.
 
 |===
 
-In normal conditions, the Workflow will transition from `Running`, `WaitingForDeployment` condition to `Running`. In case something wrong happens, consult the section xref:cloud/operator/developing-workflows.adoc#troubleshooting[Workflow Troubleshooting in Development].
+In normal conditions, the Workflow will transition from `Running` to `WaitingForDeployment`and to `Running` condition. In case something wrong happens, consult the section xref:cloud/operator/developing-workflows.adoc#troubleshooting[workflow troubleshooting in development mode].
```

Comment:
```suggestion
In normal conditions, the Workflow will transition from `Running` to `WaitingForDeployment`and to `Running` condition. In case something wrong happens, consult the section xref:cloud/operator/developing-workflows.adoc#troubleshooting[Workflow Troubleshooting in development mode].
```